code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -48,77 +49,85 @@
 	desc = "A fragmentation grenade produced for private security firms. It explodes 3 seconds after the pin has been pulled."
 	icon_state = "grenade_pmc"
 	item_state = "grenade_ex"
+	hud_state = "grenade_frag"
 	underslug_launchable = FALSE
 
+/obj/item/explosive/grenade/frag/PMC/prime()
+	spawn(0)
+		explosion(loc, -1, -1, 4)
+		cdel(src)
+	return
 
-	prime()
-		spawn(0)
-			explosion(loc, -1, -1, 4)
-			cdel(src)
-		return
```

Comment:
don't do this unless you're doing a reformatting pr because it makes it more time consuming to review

Reply:
In this case the code hasn't changed, and the code cleanup is positive.

Reply:
I'm assuming there are code changes in this file, though.

Reply:
Yes there are.